hackathon_site/event/jinja2/event/dashboard_base.html

```diff
@@ -1,194 +1,192 @@
+                            href="{{ url("registration:leave-team") }}"
+                            class="btn-small waves-effect waves-light colorBtn"
+                            style="margin-right: auto; margin-top: 15px;"
+                        >
+                            Leave team
+                        </a>
+                        <br />
+                        <br />
+                        <h3 class="formH3">Join a different team</h3>
+                        <form class="valign-wrapper" style="margin-left: 0;" method="post">
+                            {{ csrf_input }}
+                            <div class="input-field">
+                                {{ join_team_form.team_code }}
+                                {{ join_team_form.team_code.label_tag() }}
+
+                                {% if join_team_form.team_code.errors %}
+                                    <span class="formFieldError">
+                                    {% for error in join_team_form.team_code.errors %}
+                                        {{ error }}
+                                        <br />
+                                    {% endfor %}
+                                    </span>
+                                {% endif %}
+                            </div>
+                            <div style="margin-left: 20px;">
+                                <button class="btn-small waves-effect waves-light secondaryColor colorBtn center" type="submit">Submit</button>
+                            </div>
+                        </form>
+                    {% endif %}
 
-            <p class="faqQuestion">How does the waitlist work the day of?</p>
-            <p class="faqAnswer">The rank on the waitlist is first-come first serve, meaning the sooner you arrive at the event on Saturday, the higher your placement on our waitlist. At 10AM, if we have any spots remaining, we will be letting waitlisted people in.</p>
+                {% endif %}
+            </div>
+            {% endif %}
 
-            <p class="faqQuestion">Other questions?</p>
-            <p class="faqAnswer">If you have any questions or concerns, feel free to contact us at <a href="mailto:{{ contact_email }}" class="primaryText hoverLink">{{ contact_email }}</a>.</p>
+            <div class="borderTopDiv z-depth-1">
+                <h2 class="formH2" id="faq">Application FAQs</h2>
+
+                <p class="faqQuestion">When can I expect to hear back?</p>
+                <p class="faqAnswer">Acceptances will be sent out on a rolling basis, so apply early, and you will hear back sooner!</p>
+                
+                <p class="faqQuestion">Do I need to be in a team? What can I do if I don’t have a team?</p>
+                <p class="faqAnswer">No, you don’t have to be in a team! In fact, we only accept individual applications from applicants. However, feel free to form a team with other accepted applicants and collaborate on the same project.</p>
```

Comment:
Just to be clear, are we allowing people to submit individual projects? Or is this question only about being in a team for applying.

Reply:
yes, if they are in a team of 1

Reply:
I made in more clear in  What size of team is allowed?

Reply:
Looks good!